extensions/wikia/VideoHandlers/VideoFileUploader.class.php

```diff
@@ -163,11 +152,37 @@ public function upload( &$oTitle ) {
 
 		wfRunHooks('AfterVideoFileUploaderUpload', array($file, $result));
 
-		wfProfileOut(__METHOD__);
 		return $result;
 	}
 
 	/**
+	 * Get the normalized composed version of the title
+	 *
+	 * @return string
+	 */
+	public function getNormalizedDestinationTitle() {
+		return \UtfNormal::toNFC( $this->getSanitizedTitleText() );
```

Comment:
Wow, after all that it basically boils down to this one line, huh? Way to stick with this ticket :)